addons/mail/models/mail_render_mixin.py

```diff
@@ -105,13 +105,17 @@ def write(self, vals):
         return True
 
     def _update_field_translations(self, fname, translations, digest=None):
+        installed_lang = list(code for code, _ in self.env['res.lang'].get_installed())
```

Comment:
This should not be done here. Mail just add a hook to check if the current user has the rights to update a translation, as it depends on QWeb rights.

Reply:
Still thinking it should be done at 'super' level, not sure to see why mail is involved here. It is just an override, unless there is something I did not understand ?